spec/services/user_signup_spec.rb

```diff
@@ -51,18 +55,18 @@
 
     context "when personal info is valid but card is invalid" do
       it "does not create a user" do
-        charge = double('charge', successful?: false, error_message: 'Your card was declined.')
-        allow(StripeWrapper::Charge).to receive(:create).and_return(charge)
+        customer = double('charge', successful?: false, error_message: 'Your card was declined.')
+        allow(StripeWrapper::Customer).to receive(:create).and_return(customer)
         UserSignup.new(Fabricate.build(:user)).sign_up('123', nil)
```

Comment:
It looks strange if you have to pass `nil` to a method like that. Make a default nil for second argument.